modules/rosa-accessing-your-cluster.adoc

```diff
@@ -121,18 +122,21 @@ Created:     May 27, 2020
 +
 [source,terminal]
 ----
-$ oc login --token=z3sgOGVDk0k4vbqo_wFqBQQTnT-nA-nQLb8XEmWnw4X --server=https://api.rh-rosa-test-cluster1.j9n4.s1.devshift.org:6443
+$ oc login --token=z3sgOGVDk0k4vbqo_wFqBQQTnT-nA-nQLb8XEmWnw4X --server=https://api.rh-rosa-test-cluster1.j9n4.s1.devshift.org:6443 #<1>
```

Comment:
You can remove # from [source,terminal] samples as it is not used with call outs in [source,terminal].

See: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks